apps/test/unit/lib/kits/maker/boards/circuitPlayground/CircuitPlaygroundBoardTest.js

```diff
@@ -20,6 +20,7 @@ import {
   CIRCUIT_PLAYGROUND_PORTS,
```

Comment:
what is the conceptual difference between `CircuitPlaygroundBoardTest` and `CircuitPlaygroundMakerBoardTest` (and the analogous micro:bit files)?

Reply:
I sat with this naming for a while and am not super satisfied with the outcome so thank you for bringing it up. These names original came from the CPMBTest being the test that were in the MakerBoardTest that were CP specific. The CPMBTest actually covers the CP specific components and checks that they are available from the MB interface. The CPComponentsTest tests the initialization/resetting/availability of the components. Since this file exports a function to be run, would renaming the file something like "testComponentsAreAvailableFromInterpreter" be a better name?

Reply:
My instinct is to merge the contents of CPMBTest into CPBTest, and MBMBTest into MBBTest.

My original goal for `MakerBoardTest` was providing a way (in untyped JavaScript) to check that `CircuitPlaygroundBoard`, `MicroBitBoard`, `FakeBoard` and any other future boards all conform to some common `MakerBoard` interface.  Given the board components are no longer part of that common interface, it seems reasonable to test them with the specific board type.  (Of course it's fine if you wanted separate test files just to keep things organized, but then maybe it should be `circuitPlayground/componentsTest.js` or something.)

Reply:
It is a bit confounding that `FakeBoardTest` calls the CP version of `itMakesComponentsAvailableFromInterpreter` and so you do want to make that available externally - that messes with names a bit.  It makes me wonder if `FakeBoard` makes sense at all anymore - maybe it should explicitly be `FakeCircuitPlaygroundBoard.`

Reply:
that totally makes sense. +1 to Brad's suggestion to either 1) move CPMBTest into CPBTest, or 2) rename CPMBTest to componentsTest if having all tests together is confusing or feels strange. 

the thing i like about having separate test files here is it makes the goal very intention if i were to add a new board. i would look at the other directories, see `componentsTest.js` and realize that this new board should also test its interface in the same way.

Reply:
I ended up merging the CPMB and CPB tests - I think that streamlines things nicely. I also tried to be clear the the FakeBoard was calling CP test, but I held off on renaming the FakeBoard now. Thanks for the feedback y'all!